regression-tests/test-results/gcc-13/mixed-bounds-check.cpp.execution

```diff
@@ -1,2 +1 @@
 Bounds safety violation: out of bounds access attempt detected
-terminate called without an active exception
```

Comment:
This is the platform-dependent message (c.f. the one on macOS above) removed thanks to std::set_terminate(std::abort);.